src/frontend/src/components/groups/GroupRow/GroupRow.tsx

```diff
@@ -0,0 +1,117 @@
+import { Avatar, Box, Card, CheckBox, Grid, Text } from 'grommet';
+import React, { ChangeEvent, useRef, useState } from 'react';
+import { More } from 'grommet-icons';
+import { useMediaQuery } from 'react-responsive';
+
+export interface Group {
+  id: string;
+  name: string;
+  members: {
+    id: string;
+    name: string;
+    avatar: string;
+  }[];
```

Comment:
`members` can be extracted as a single interface ? Then you use it in you `Group` interface.

```suggestion
  members: Array<{
    id: string;
    name: string;
    avatar: string;
  }>;
```

Reply:
In your opinion, where such interfaces should go ? In the `/src/types`, (very global), in a `/src/components/groups/types.ts` (root of where we need it, but it adds files randomly), or in one of the components that use it, as it is right now...